### DISCUSSION TITLE
Finding an orientation of a grain based on another grain

### DISCUSSION THREAD

**Discussion by danishTUE** [mtex-toolbox/mtex#2198] [Ask Anything] [answered]
Hello Everyone,

I am new to MTex, so pardon if the query is very trivial. 

I have a FCC grain (ley say _g1_) with a certain orientation (ley say with Bunge Euler Angle (in degrees): 21.46 156.22 245.5). I am interested in finding the orientation of another grain (ley say _g2_) such that the plane **(111)** of _g1_ is parallel to plane **(-1-11)** of _g2_ in a way that directions **[ 0 1-1]**, **[-1 0 1]** and **[1-1 0]** of _g1_ are parallel to the directions  **[0 -1 -1]** **[1 0 1]** **[-1	1 0]** of _g2_. I see from the documentation that probably _rotation.map_ or _rotation.fit_ should be used for this purpose. However, I am not very confident about it. Can you kindly suggest, how can I obtain the orientation of _g2_ using MTex?

Regards,

Danish

**Comment by ralfHielscher** [comment, marked as answer]
You are welcome
```matlab
cs = crystalSymmetry('432');
g1 = orientation.byEuler([21.46 156.22 245.5]*degee,cs)
mori = orientation.map(Miller(1,1,1,cs),Miller(-1,-1,1,cs),Miller(0,1,-1,cs,'uvw'),Miller(0,-1,-1,cs,'uvw')))
g2 = g1 * inv(mori)
```
Ralf.

**Reply by danishTUE** [reply]
Many thanks Dr. Ralf. That worked well. I think in this way we can also calculate the orientation of a martensite variant (BCC) from parent austenite orientation (FCC). However, I do see that there is a dedicated MTex functionality for that.

**Reply by nyyssont** [reply]
Please see the examples in this section:
https://mtex-toolbox.github.io/MartensiteVariants.html

Modifying the command by Ralf:
`KS = orientation.map(Miller(1,1,1,cs_aus),Miller(0,1,1,cs_bcc), Miller(-1,0,1,cs_aus),Miller(-1,-1,1,cs_bcc))`

This produces the Kurdjumov-Sachs orientation relationship and is taken straight from the linked examples.
